print/src/main.rs

```diff
@@ -106,6 +121,27 @@ impl Source {
         Ok(Source::Serial(ser))
     }
 
+    async fn set_rtt_addr(&mut self, elf_bytes: &[u8]) -> anyhow::Result<()> {
+        let Source::Tcp(tcpstream) = self else {
```

Comment:
Since passing --set-addr only makes sense with tcp we could warn if it is used with serial. Or we could move it into Command::Tcp so it is only available with tcp.

Reply:
Good point - fixed.